File: Data Structure/Lab 3/Duplicate.c
// Write a program to find whether the array contains a duplicate number or not.

#include<stdio.h>
void main(){
    int n,a[n],i = 0,j = 0;

    printf("Enter size of an array(n) :");
    scanf("%d",&n);

    for(i = 0; i < n ; i++){
        printf("Enter element :");
        scanf("%d",&a[i]);
    }

    for(i = 0; i < n; i++){
        for(j = i+ 1; j < n + 1;j++){
            if(a[j] == a[j + 1]){
                printf("Duplicate");
                return;
            }
        }
    }
}

File: Data Structure(Java)/Lab 2/Duplicate.java
// Write a program to find whether the array contains a duplicate number or not.

import java.util.Scanner;

public class Duplicate {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter size of an array :");
        int n = sc.nextInt();
        int[] arr = new int[n];

        for(int i = 0; i < n ; i++){
            System.out.println("Enter array element at index "+i);
            arr[i] = sc.nextInt();
        }

        for(int i = 0; i < n; i++){
            for(int j = 1 ; j < n ; j++){
                if(arr[i] == arr[j]){
                    System.out.println("Duplicate !");
                    sc.close();
                    return;
                }
            }
        }
        sc.close();
    }
}
